File: js/medium/compareStr.js
function compare(strA, strB) {
  if (strA <= strB) {
    return strA;
  } else {
    return strB;
  }
}
console.log(compare('banana', 'avocado')); //  'avocado'

function compare1(strA, strB) {
  var result = strA.localeCompare(strB, {
    ignorePunctuation: true,
    sensitivity: 'base',
  });
  return result > 0 ? strB : strA;
}
console.log(compare1('banana', 'avocado')); //  'avocado'
console.log(compare1('Banana', 'avocado')); //  avocado

File: python/medium/compare_str.py
def compare(a,b):
  if a.upper() <= b.upper():
    return a
  else:
    return b
print(compare('banana', 'avocado'))
print(compare('Banana', 'avocado'))

class Compare():
    def __init__(self):
        pass

    def two_strs(self, a, b):
        if a.lower() <= b.lower():
            return a
        else:
            return b
test = Compare()
print(test.two_strs('banana','avocado'))
print(test.two_strs('Banana','avocado'))
print(test.two_strs('Awesome','Avocado'))
